<?xml version="1.0"?>
<launch>

  <!-- <node pkg="nodelet" type="nodelet" name="hdl_people_detection_nodelet" args="load hdl_people_detection/PersonDetectorNode $(arg nodelet_manager)"> -->
  <node pkg="person_tracking" type="person_det_node" name="hdl_people_detection_nodelet" >
  	<param name="static_sensor" value="true"/>
    <param name="downsample_resolution" value="0.1"/>
    <param name="backsub_resolution" value="0.2"/>
    <param name="backsub_occupancy_thresh" value="2"/>
    <param name="cluster_min_pts" value="20"/>
    <param name="cluster_max_pts" value="180"/>
    <param name="cluster_min_size_x" value="0.2"/>
    <param name="cluster_min_size_y" value="0.2"/>
    <param name="cluster_min_size_z" value="-1.4"/>
    <param name="cluster_max_size_x" value="1.0"/>
    <param name="cluster_max_size_y" value="1.0"/>
    <param name="cluster_max_size_z" value="2.0"/>
    <param name="enable_classification" value="true"/>
  </node>

  <node pkg="person_tracking" type="person_tracking_node" name="person_tracking_node">
    <param name="remove_trace_thresh" value="1.0"/>
    <param name="human_radius" value="0.4"/>
  </node>
</launch>
